columnflow/production/normalization.py

```diff
@@ -331,6 +340,13 @@ def normalization_weights_init(self: Producer) -> None:
     Initializes the normalization weights producer by setting up the normalization weight column.
     """
     self.produces.add(self.weight_name)
+    if self.allow_stitching:
+        self.stitching_datasets = self.get_stitching_datasets()
+    else:
+        self.stitching_datasets = [self.dataset_inst]
+
+    if self.allow_stitching and self.dataset_inst == self.get_inclusive_dataset():
+        self.produces.add(f"{self.weight_name}_inclusive_only")
```

Comment:
Picky, but while we're on it, better store the inclusive weight name as an attribute as well, i.e.,

```python
self.weight_name_incl = f"{self.weight_name}_inclusive_only"
```

and then use it above.